en/flight_controller/intel_aero.md

```diff
@@ -93,3 +93,16 @@ The pinout for the Lidar Lite V3 is as follows
 ![](../../assets/hardware/Aero_I2C_splitter.JPG)
 
 ![](../../assets/hardware/Aero_LidarLite.JPG)
+
+## Using Optical Flow on the Aero
+
+The Intel Aero comes with a preinstalled optical flow binary (OS version 1.6 or higher) which enables the Aero to stably fly based on vision. In order to use optical flow, a range sensor has to be installed first (see above).
```

Comment:
This is the user guide - if you say "OS" users are likely to assume PX4 version (we're currently on 1.6.x) Is this what you mean? In that case I'd change to "PX4 v1.6". If not, specify the OS.

Reply:
No, it's the Linux verison